packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -279,6 +280,51 @@ export default class VTable {
         this.forEachCellOfColumn(this.col, callback);
     }
 
+    /**
+     * Loop each table cell and get all the cells that share the same border.
+     * The result is an array of table cell elements where the first element is the narrowest td
+     * @param borderPos The position of the border
+     * @param isRightBorder Indicate whether it is a right border or left border
+     */
+    getCellsWithBorder(borderPos: number, isRightBorder: boolean): HTMLTableCellElement[] {

```

Comment:
Better add more comments inside code, and add an example

Reply:
added comments